DSC-Unilag/DSC-Blog
# Initialize firebase project

#### What does this PR do?
- This PR sets up firebase cloud functions on the application
#### Description of Task to be completed?
-  Setup firebase cloud functions
- Add config examples
- Add eslint
#### How should this be manually tested?
- Pull this branch [daniellamarr:initialize-firebase-project](https://github.com/daniellamarr/DSC-Blog/tree/initialize-firebase-project)
- On a terminal, run `cd functions` to enter into the `functions` folder
- Install all dependencies by running `npm install`
- Run `npm run serve` to start the local server
- On a web browser, navigate to the local server `/welcome` to see the welcome page
#### What are the relevant issues?
N/A
#### Screenshots (if appropriate)
N/A
